ideas.rb

```diff
@@ -0,0 +1,52 @@
+class Takeway
+  def initialize(menu = Menu.new, basket = Basket.new, text_providor = TextProvidor.new)
```

Comment:
Maybe you could find another way to link your classes, as I think best practice is that each class works and can be tested separately without relying on each other